.gitignore

```diff
@@ -99,10 +99,84 @@ META
 /lex/ocamllex
 /lex/ocamllex.opt
 /lex/parser.output
-
+/manual/src/allfiles.tex
+/manual/src/biblio.tex
+/manual/src/foreword.tex
+/manual/src/version.tex
+/manual/src/warnings.etex
+/manual/src/warnings.tex
+/manual/src/foreword.htex
+/manual/src/manual.html
+/manual/src/webman
+/manual/src/ifocamldoc.tex
+/manual/src/cmds/*.tex
+/manual/src/cmds/*.htex
+/manual/src/cmds/warnings.etex
 /manual/src/cmds/warnings-help.etex
+/manual/src/html_processing/.sass-cache
+/manual/src/html_processing/camlp-streams
+/manual/src/html_processing/dune
+/manual/src/html_processing/markup.ml
+/manual/src/html_processing/uchar
+/manual/src/html_processing/uutf
+/manual/src/html_processing/lambdasoup
+/manual/src/html_processing/ocaml-re
 /manual/src/html_processing/src/common.ml
+/manual/src/htmlman/*.html
+/manual/src/htmlman/*.haux
+/manual/src/htmlman/*.hind
+/manual/src/htmlman/compilerlibref
+/manual/src/htmlman/highlight.pack.js
+/manual/src/htmlman/libref
+/manual/src/htmlman/manual.hmanual
+/manual/src/htmlman/manual.hmanual.kwd
+/manual/src/htmlman/manual.css
+/manual/src/htmlman/odoc.css
+/manual/src/htmlman/*.htoc
+/manual/src/htmlman/*.svg
+/manual/src/infoman/*.haux
+/manual/src/infoman/*.hind
+/manual/src/infoman/*.info*.gz
+/manual/src/infoman/*.info.body*
+/manual/src/infoman/ocaml.hocaml.kwd
+/manual/src/library/*.tex
+/manual/src/library/*.htex
+/manual/src/library/arithstatus.mli
```

Comment:
The `arithstatus.ml` file is a very old import from the csv ignore file which is no longer relevant.